plugins/modules/btrfs_info.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/python
+
+# Copyright (c) 2022, Gregory Furlong <gnfzdz@fzdz.io>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: btrfs_info
+short_description: Returns the current status of all identified btrfs filesystems
```

Comment:
I'm a bit confused what the official spelling of btrfs/Btrfs/BTRFS is. WikiPedia sticks to Btrfs, the wiki at https://btrfs.wiki.kernel.org/index.php/Main_Page uses both Btrfs and btrfs, and the official documentation at https://btrfs.readthedocs.io/en/latest/ uses BTRFS. :shrug: Do you have any insight which of these is the best variant to use when referring to the filesystem (and not the CLI program, that's always lowercase `btrfs`)?

Reply:
As another example the man pages seem to consistently use lowercase btrfs.  I'm actually not sure if one is preferred officially, but I'll try to find out. For now, the modules consistently use lowercase btrfs.